beta-prompts/beta_prompts/visualizations.py

```
@@ -0,0 +1,1233 @@
+        )
+        plt.close()
+
+        return output_file
+
+    def _create_performance_evolution(self, ax, data: VisualizationData):
+        """Performance evolution with gradient fills and trend indicators"""
+
+        # Generate sample data if not provided
+        sessions = data.optimization_sessions or self._generate_sample_sessions()
+
+        dates = [datetime.now() - timedelta(days=30 - i) for i in range(len(sessions))]
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)